src/loaders/api/api.js

```diff
@@ -194,6 +195,14 @@ export function setAPI (agentIdentifier, forceDrain, runSoftNavOverSpa = false)
   function apiCall (prefix, name, notSpa, bufferGroup) {
     return function () {
       handle(SUPPORTABILITY_METRIC_CHANNEL, ['API/' + name + '/called'], undefined, FEATURE_NAMES.metrics, instanceEE)
+      dispatchGlobalEvent({
+        agentIdentifier,
+        loaded: true,
```

Comment:
I think after you pointed out the flakiness of the tests, the `loaded` attribute here (and maybe in all the emitters needs to be "smarter".  It could probably just check to see if the exported `activatedFeatures` object has been set yet -- if no, loaded = `false`, if yes, loaded = `true`.   The API can technically be called before the window/agent has fully loaded.

Reply:
this could be improved in a future ticket, because i think there are some things we could do to unify all the dispatch calls across the agent to provide common attributes (agentIdentifier, loaded) without requiring the individual modules to pass that stuff around so much -- like make the dispatcher a higher order function stored in the runtime or something.  I'll add a separate ticket for that work that can extend from this as to not derail this work